packages/harmony/src/storybook/components/ColorPalette/ColorPalette.tsx

```diff
@@ -44,25 +45,30 @@ const ColorRow = ({ colors, themeKey }: ColorRowProps) => {
   const colorNames = Object.keys(colors)
 
   return (
-    <section className={styles.colorSection}>
-      <div className={styles.colorSectionInfo}>
+    <Flex gap='4xl'>
+      <Flex className={styles.colorSectionInfo} direction='column' gap='l'>
```

Comment:
this is awesome!

Reply:
what do you think about `col` and `row` being top level boolean props?

`<Flex column gap='l' />`

Reply:
Seconded this idea! 👍 Might be nice to have a quick shorthand

Reply:
Excellent point. I suspect row and column will be the most common, but we'll need others too. You could support all the options as bool props, but that gets a bit silly, and the types get weird.

What about `direction='row'` as the default? Maybe not worth sacrificing readability just to save characters. Generally, I'd like the `<Flex ... />` and `<Box ... />` to fit on one line for readability, so favoring short props and optionals